apps/mobile/src/screens/Authentication/CreatePasscode.tsx

```diff
@@ -1,11 +1,10 @@
 import { CreatePasscode } from '@walless/app';
+import { appActions } from 'state/app';
 import { navigate } from 'utils/navigation';
 
-import { initLocalDeviceByPasscodeAndSync } from '../state/app/authentication';
-
 export const CreatePasscodeScreen = () => {
 	const handleOnComplete = async (passcode: string) => {
-		await initLocalDeviceByPasscodeAndSync(passcode);
+		await appActions.signInWithPasscode(passcode);
```

Comment:
meaning of this method is setup keys, encryption (in a local device) after google sign in phase. It might cause confuse to use 'signIn' again

Reply:
using `signInWith...` looks like sign in by different methods than stages. Otherwise, with deprecated passcode, action to use passcode is actually 'sign in' by look up `tkey question module`. But using 'passcode' to setup local encryption might be not sign in (still a stage).